junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -889,6 +892,27 @@ public static Try<Class<?>> tryToLoadClass(String name, ClassLoader classLoader)
 		});
 	}
 
+	/**
+	 * Tries to load the {@link Resource} for the supplied classpath resource name.
+	 *
+	 * <p>See {@link org.junit.platform.commons.support.ReflectionSupport#tryToLoadResource(String, ClassLoader)}
+	 * for details.
+	 *
+	 * @param classpathResourceName the name of the resource to load; never {@code null} or blank
+	 * @param classLoader the {@code ClassLoader} to use; never {@code null}
+	 * @since 1.11
+	 */
+	@API(status = INTERNAL, since = "1.11")
+	public static Try<Resource> tryToLoadResource(String classpathResourceName, ClassLoader classLoader) {
+		Preconditions.notBlank(classpathResourceName, "Resource name must not be null or blank");
+		boolean startsWithSlash = classpathResourceName.startsWith("/");
+		URL resource = classLoader.getResource(startsWithSlash ? "/" + classpathResourceName : classpathResourceName);
```

Comment:
Why are we adding another slash if the resource name already starts with one?
```suggestion
		URL resource = classLoader.getResource(startsWithSlash ? classpathResourceName : "/" + classpathResourceName);
```

Reply:
Mmh. Looks like I dropped the tests when pulling this in from #3718.

Reply:
Okay. Looks like I got the definition the wrong way around. The leading `/` should be removed prior to class loading because we're using `ClassLoader.getResource()` and not  `Class.getResource()`. Derp.